boot/modeenv.go

```diff
@@ -62,6 +62,13 @@ type Modeenv struct {
 	Model               string   `key:"model"`
 	BrandID             string   `key:"model,secondary"`
 	Grade               string   `key:"grade"`
+	// BootFlags is the set of boot flags. Whether this applies for the current
```

Comment:
I was confused by the "whether..." sentence.  Maybe we should write instead something like:

```
BootFlags is the set of boot flags. The flags that apply to the current boot should be retrieved with the BootFlags function. Initramfs reads this field when applicable and captures the flags to a file in /run that userspace reads instead of this key. When setting...
```

Reply:
done